tests/languages/xojo/keyword_feature.test

```diff
@@ -108,116 +108,7 @@ WString
 
 ----------------------------------------------------
 
-[

```

Comment:
That is step 1. Now you need to run `npm run test:languages -- --accept`. This will insert the current token stream in here.